test/framework/toy_build.py

```diff
@@ -3032,28 +3032,34 @@ def end_hook():
                print('end hook triggered, all done!')
 
             def pre_run_shell_cmd_hook(cmd, *args, **kwargs):
-                if cmd.strip() == TOY_COMP_CMD:
+                if isinstance(cmd, str) and cmd.strip() == TOY_COMP_CMD:
                     print("pre_run_shell_cmd_hook triggered for '%s'" % cmd)
                     # 'copy_toy_file' command doesn't exist, but don't worry,
                     # this problem will be fixed in post_run_shell_cmd_hook
                     cmd += " && copy_toy_file toy copy_of_toy"
-                return cmd
+                    return cmd
```

Comment:
```suggestion
                return cmd
```

Indentation has changed comparing to https://github.com/easybuilders/easybuild-framework/blob/develop/test/framework/toy_build.py#L2941

Reply:
I had a reason to do this, but it was probably due to a bug that's now fixed in the `run` function, so it should indeed be no longer needed, I'll check it...